arch/xtensa/src/esp32/esp32_wifi_adapter.c

```diff
@@ -3960,6 +4088,47 @@ static unsigned long esp_random_ulong(void)
   return esp_random();
 }
 
+/****************************************************************************
+ * Name: esp_wifi_tx_done_cb
+ *
+ * Description:
+ *   WiFi TX done callback function.
+ *
+ ****************************************************************************/
+
+static IRAM_ATTR void esp_wifi_tx_done_cb(uint8_t ifidx, uint8_t *data,
+                                          uint16_t *len, bool txstatus)
+{
+#if 0
+  wlinfo("INFO: ifidx=%d data=%p *len=%p txstatus=%d\n",
+         ifidx, data, len, txstatus);
+#endif
```

Comment:
About this, because I also don't want to enable this debugging unless it is necessary. Here will be called when packet id RX done, so this will lead to too much log.